test/tst_netcdftime.py

```diff
@@ -699,5 +699,70 @@ def test_dayofwk(self):
 
             old_date = date
 
+class issue593TestCase(unittest.TestCase):
```

Comment:
Better to use a sensible name than a meaningless issue number, unless it's for a specific regression test.